dashboards-observability/public/components/metrics/index.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiSpacer,
+  EuiSuperDatePicker,
+  EuiSuperDatePickerProps,
+  EuiText,
+  EuiTitle,
+  EuiTabbedContent,
+  OnRefreshChangeProps,
+  OnRefreshProps,
+  OnTimeChangeProps,
+  ShortDate,
+  EuiPageContentBody,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { DurationRange } from '@elastic/eui/src/components/date_picker/types';
+import React, { Fragment, useState } from 'react';
+import { StaticContext } from 'react-router-dom';
+import { Route, RouteComponentProps } from 'react-router-dom';
+import { uiSettingsService } from '../../../common/utils';
+import { ChromeBreadcrumb, CoreStart } from '../../../../../src/core/public';
+import { ObservabilitySideBar } from '../common/side_nav';
+import { onTimeChange } from './helpers/utils';
+
+interface MetricsProps {
+  http: CoreStart['http'];
+  chrome: CoreStart['chrome'];
+  parentBreadcrumb: ChromeBreadcrumb;
+  renderProps: RouteComponentProps<any, StaticContext, any>;
+}
+
+export const Home = ({
+  http,
+  chrome,
+  parentBreadcrumb,
+  renderProps,
+}: MetricsProps) => {
+  const [recentlyUsedRanges, setRecentlyUsedRanges] = useState<DurationRange[]>([]);
+  const [start, setStart] = useState<ShortDate>('now-30m');
+  const [end, setEnd] = useState<ShortDate>('now');
+  const [dateDisabled, setDateDisabled] = useState(false);
+
+  const onRefreshFilters = (startTime: ShortDate, endTime: ShortDate) => {
+    // if (!isDateValid(convertDateTime(startTime), convertDateTime(endTime, false), setToast)) {
+    //   return;
+    // }
+
+    console.log('refreshed date picker');
+  };
+
+  const onDatePickerChange = (props: OnTimeChangeProps) => {
+    onTimeChange(
```

Comment:
if onTimeChange were included in this component's code, it's usage would be clear, you would be able to reduce the args of the Fn (setRCU, setStart, setEnd are in-context).  This callback (onDatePickerChange) could have been reduced to :

(props) => {
  onTimeChange(props);
  onRefreshFilters(props);
}

onRefreshFilters would need a simple refactor to destructure the props instead of taking args-list.